src/Firebase.cpp

```diff
@@ -53,11 +61,45 @@ FirebaseGet Firebase::get(const String& path) {
   return FirebaseGet(host_, auth_, path, &http_);
 }
 
+FirebaseSet Firebase::set(const String& path, int value) {
+  return FirebaseSet(host_, auth_, path, jsonEncode(value), &http_);
+}
+FirebaseSet Firebase::set(const String& path, float value) {
+  return FirebaseSet(host_, auth_, path, jsonEncode(value), &http_);
+}
+FirebaseSet Firebase::set(const String& path, double value) {
+  return FirebaseSet(host_, auth_, path, jsonEncode(value), &http_);
+}
 FirebaseSet Firebase::set(const String& path, const String& value) {
```

Comment:
Note: that this will break client that were pushing json encoded string.

If needed, we could error out if the `value` is decoding to a valid json object / literal to help with the transition.